README.md

```diff
@@ -34,6 +34,42 @@ To stop the docker environment, run:
   docker-compose down
   ```
 
+## Adding a rule
+
+To perform a very basic end-to-end test of StackStorm, let's create a simple rule.
+Run the following from your docker host.
+
+```
+cp -R examples packs.dev
+```
+
+First, let's run `st2 login`:
+
+```
+docker exec eval_st2_1 st2 login -w -p Ch@ngeMe st2admin
```

Comment:
1) Where this `eval_st2_1` container name came up? It's no good.
2) Didn't we place .st2/config file in place so we don't need to login any more?